### ISSUE TITLE
Form 21P-601: Accessibility issues -Incorrect heading structure

### ISSUE THREAD

**Issue by nkutub** [department-of-veterans-affairs/va.gov-team#127780]
## BACKGROUND
Team BIO-HEART completed Staging Review for Form 21P-601 on 10/30/25. There were 38 total findings identified by VA Platform (8 were launch blocking, while 30 were non-launch blocking). 

## PROBLEM STATEMENT / GOAL

- Page: Review your expenses
- Device: All
- Orientation: All 
The alert uses an H3 and the legend uses an H4. This implies that the legend “belongs” to the alert, or is a subsection of the alert, which isn’t accurate.

<img width="808" height="532" alt="Image" src="https://github.com/user-attachments/assets/092d568b-a9a9-4731-8033-da192d870ac1" />

**Issue Severity**
- Expected WCAG failure: n/a (not a WCAG issue)
 - Expected 508 audit severity: n/a
- Expected CC severity level: possibly a11y-defect-2 (HTML should have properly nested headings) - there isn’t a skipped heading level so it might be a11y-defect-4
- Expected experience standard: User encounters content that meets best practices

**How to recreate:**
-Fill out the form until you get to the expenses list-and-loop
- Add an expense to get to “Review your expenses”
- View the source code or use the HeadingsMap extension to view the semantic headings

**Recommendation:** 
- Short-term fix: Make the legend an H3 instead of an H4.
- Long-term fix: I’m not sure the alert needs to be here, since it was already present before adding an expense. Consider removing it from this page.
